README.md

```diff
@@ -28,52 +28,70 @@ Before you proceed, make sure you have the following software installed:
     On Debian/Ubuntu:
     
     ```bash
-    $ sudo apt install libpq-dev
+     sudo apt install libpq-dev
     ```
+ * Cargo dependency
+ * 
+   On Debian/Ubuntu:
+    
+    ```bash
+     sudo apt install cargo
+    ```
+    
+### Prepare Database
 
+Setup PostgreSQL database
 
-### Prepare Database
+ ```bash
+ sudo apt-get install postgresql postgresql-contrib
+```   
 
-Setup PostgreSQL database, create a database with the regular tools, and note the connection string (database host, credentials, and the database name).
+Create a database with the regular tools (Change user and db_name with yours)
+    ```bash
+     sudo su postgres
+     psql
+     create database db_name with owner=user;
+    ```
+Note the connection string (database host, credentials, and the database name).
 
 Clone this repository and open the project folder
 
 ```bash
-$ git clone git@github.com:near/near-indexer-for-explorer.git
-$ cd near-indexer-for-explorer
+ git clone https://github.com/near/near-indexer-for-explorer.git
+ cd near-indexer-for-explorer
 ```
 
 You need to provide database credentials in `.env` file like below (replace `user`, `password`, `host` and `db_name` with yours):
 
 ```bash
-$ echo "DATABASE_URL=postgres://user:password@host/db_name" > .env
+ echo "DATABASE_URL=postgres://user:password@host/db_name" > .env
 ```
 
 Then you need to apply migrations to create necessary database structure. For this you'll need `diesel-cli`, you can install it like so:
 
 
 ```bash
-$ cargo install diesel_cli --no-default-features --features "postgres"
+ cargo install diesel_cli --no-default-features --features "postgres"
 ```
 
 And apply migrations
 
 ```bash
-$ diesel migration run
+ diesel migration run
 ```
 
 ### Compile NEAR Indexer for Explorer
 
 ```bash
-$ cargo build --release
+ cargo build --release
 ```
 
 ### Configure NEAR Indexer for Explorer
 
 To connect NEAR Indexer for Explorer to the specific chain you need to have necessary configs, you can generate it as follows:
 
 ```bash
-$ ./target/release/near-indexer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
+ ./target/release/near-indexer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
```

Comment:
As I see in current version the `./target/release/near-indexer` is renamed into `./target/release/indexer-explorer`